.github/workflows/ci.yaml

```diff
@@ -195,13 +201,21 @@ jobs:
             env:
               RUST_LIBC_UNSTABLE_GNU_FILE_OFFSET_BITS: 64
             artifact-tag: offset-bits64
+          - target: arm-unknown-linux-gnueabihf
+            env:
+              RUST_LIBC_UNSTABLE_GNU_TIM_BITS: 64
```

Comment:
Just noticed the misspelling here.  I will push a rebase as soon as I've verified a build with the proper variable.

Reply:
I've pushed an updated version now.